<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistWidget</class>
 <widget class="QWidget" name="PlaylistWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>258</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>29</number>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">#</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Channel</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="searchEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Clear</string>
     </property>
     <property name="icon">
	  <iconset resource="../../../data/icons.qrc">
       <normaloff>:/icons/32x32/clear.png</normaloff>:/icons/32x32/clear.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="categoryBox">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelSearch">
     <property name="text">
      <string>Search:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelCategory">
     <property name="text">
      <string>Category:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelLanguage">
     <property name="text">
      <string>Language:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QComboBox" name="languageBox">
     <property name="maxVisibleItems">
      <number>50</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../data/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>searchEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
